common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -191,13 +191,15 @@ def _publish_event(self, event_name, result, **kwargs):
             "result": result,
             "previous_count": getattr(self, "_last_event_result_count", len(self.selected)),
             "max_count": self.max_count,
+            "user_id": user_id,
+            "course_id": unicode(self.course_id),
```

Comment:
Even if you decide to redundantly include this data in the payload, you still need to update the code to include it in the context, I believe.